portfolio/src/main/webapp/index.html

```diff
@@ -1,32 +1,85 @@
-        <br><br />
-        <li>Joined the hackathon team for the event I previously mentioned and did some development on the website. Learned a lot about front end development (HTML, CSS, JS).</li>
-      </ul>
-      <h2>Socials
-        <h2 />
-        <a href="https://www.linkedin.com/in/connorsbrett/">
-        <img class="icon" src="https://www.pinclipart.com/picdir/big/55-557142_banner-black-and-white-download-linkedin-svg-instagram.png">
+
+<head>
+	<meta charset="UTF-8">
+	<title>Connor's Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div class="card">
+		<h1>Connor's Portfolio</h1>
+		<img class="avatar" src="https://i.imgur.com/AW18kUT.png" alt="avatar" />
+		<p>My name is Connor and I am a rising junior at University of Arizona studying Computer Science.</p>
+		<h2>Background</h2>
+		<p>I grew up in Campbell, CA and discovered my interest and love for computers at a very young age. I first
+			started playing games like Pajama Sam on the computer which then grew into a passion for video games which I
+			still have today. I took my first coding class sophomore year of high school which captured my interest in
+			CS. After taking more coding classes in high school, I continued in college and majored in Computer Science.
+			After my first year, I managed to secure an internship at a small company where I learned fundamentals of
+			software development like debugging and testing. My second year of school both exposed me to new areas of
+			computer science and pushed me harder than I had before. This combined with what I learned from my first
+			internship helped prepare me for this opportunity at Google.</p>
+		<h2>Projects</h2>
+		<ul>
+			<li>I worked on the Google Voice integration portion of a GNSS Tracking Application for a hacakthon which
+				ended up winning! I used DialogFlow to parse input from the user and send a request to a database which
+				would then report back to the user.</li>
+			<br><br />
+			<li>Joined the hackathon team for the event I previously mentioned and did some development on the website.
+				Learned a lot about front end development (HTML, CSS, JS).</li>
+		</ul>
+		<h2>Socials
+			<h2 />
+			<a href="https://www.linkedin.com/in/connorsbrett/">
+				<img class="icon" src="https://www.pinclipart.com/picdir/big/55-557142_banner-black-and-white-download-linkedin-svg-instagram.png">
         </a>
-        <a href="https://github.com/connorbrett">
-        <img class="icon" src="https://i.imgur.com/ISfB4pM.png">
+				<a href="https://github.com/connorbrett">
+					<img class="icon" src="https://i.imgur.com/ISfB4pM.png">
         </a>
-    </div>
-  </body>
-</html>
+					<h2>Slideshow
+						<h2 />
+						<div class="slideshow-container">
+							<div class="mySlides fade init">
```

Comment:
It might be better to be consistent for the style of class names (for example, you use foo-bar for slideshow-container but use fooBar for mySlides).

Also "mySlides" might not be a good class name -- does "my" matter here :) can't we just use "slides"?

Reply:
fixed var names